tools/serverpod_cli/lib/src/downloads/resource_manager.dart

```diff
@@ -149,6 +212,20 @@ class ResourceManager {
   }
 }
 
+class ServerpodCloudData {
+  late final String token;
+
+  ServerpodCloudData(this.token);
```

Comment:
Question: Do we not want to create this with one of our models?

Reply:
I would rather not do that, since we:
1. Don't currently have any generated models in the CLI.
2. Would need to create a dependency from the CLI to a serverpod package which currently doesn't exist.

Reply:
Do we not have generated models already? I thought we did, at least via dependency. 

But alright, ill buy that.

Reply:
Aah we do have models but only for the serveprod_client. So, we don't seem to have any data objects so I will not introduce it here.